mlir/utils/performance/reportUtils.py

```diff
@@ -70,7 +70,7 @@ def setCommonStyles(styler: 'pd.io.formats.style.Styler', speedupCols: list, col
         {'selector': 'tbody tr:nth-child(even)', 'props': [('background-color', '#eeeeee')]},
         {'selector': 'table', 'props': [('background-color', '#dddddd'), ('border-collapse', 'collapse')]},
         {'selector': 'th, td', 'props': [('padding', '0.5em'), ('text-align', 'center'), ('max-width', '150px')]}])
-    styler.format(precision=ROUND_DIGITS, na_rep="FAILED")
```

Comment:
Trying to print something more neutral than "FAILED" for data that we just don't have, like MIOpenDriver and LDSBankConflicts (which it doesn't report).

Reply:
Except that we do probably want our own perf results to `FAILED` if we don't successfully benchmark something but ... yeah, that's a niche case, approved

Reply:
It wants more nuance.  MIOpenDriver never reports LDSBankConflict, so that shouldn't be FAILED, but true failures should.  Meanwhile, it's weird to report a missing perfconfig as "nan".  Not solved here, or anywhere yet.